WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -245,6 +525,35 @@ - (void)configureCell:(UITableViewCell *)cell forIndexPath:(NSIndexPath *)indexP
         [self configureMostPopularCell:(InsightsMostPopularTableViewCell *)cell];
     } else if ([identifier isEqualToString:InsightsTableTodaysStatsDetailsCellIdentifier] || [identifier isEqualToString:InsightsTableTodaysStatsDetailsiPadCellIdentifier]) {
         [self configureTodaysStatsCell:(InsightsTodaysStatsTableViewCell *)cell];
+    } else if ([identifier isEqualToString:StatsTablePeriodHeaderCellIdentifier]) {
+        cell.backgroundColor = self.tableView.backgroundColor;
+        UILabel *label = (UILabel *)[cell.contentView viewWithTag:100];
```

Comment:
My goal is to refactor the cell classes to contain all of the setup logic. No more tags here eventually! :grin: